src/accordion/accordion.spec.ts

```diff
@@ -232,6 +232,35 @@ describe('ngb-accordion', () => {
     titles.forEach((title: HTMLElement, idx: number) => { expect(title.textContent.trim()).toBe(`Panel ${idx + 1}`); });
   });
 
+  it('should not pick up titles from nested accordions', () => {
+    const testHtml = `
+    <ngb-accordion activeIds="open_me">
+     <ngb-panel title="parent title" id="open_me">
+       <ng-template ngbPanelContent>
+         <ngb-accordion>
+           <ngb-panel>
+             <ng-template ngbPanelTitle>child title</ng-template>
+             <ng-template ngbPanelContent>child content</ng-template>
+           </ngb-panel>     
+          </ngb-accordion>
+       </ng-template>
+     </ngb-panel>     
+    </ngb-accordion>
+    `;
+    const fixture = createTestComponent(testHtml);
+
+    fixture.detectChanges();
```

Comment:
nitpick: kind of redundant, as `createTestComponent` already does it

Reply:
Actually the bug can only be reproduced if you run another round of change detection, so it is deliberate here.

Reply:
oh, a comment then?